content/en/agent/guide/community-integrations-installation-with-docker-agent.md

```diff
@@ -14,7 +14,7 @@ further_reading:
   text: "Agent commands"
 ---
 
-Datadog community integrations are stored in the [Integrations-extra][1] Github repository. Since they are non-officially supported by Datadog, they are not packaged with the Datadog Agent and needs to be build locally in order to be included in the Agent. Find below how to install them with your Datadog Agent running as a binary on a host or as a container.
+Datadog community integrations are stored in the [Integrations-extra][1] Github repository. Since they are non-officially supported by Datadog, they are not packaged with the Datadog Agent and needs to be built locally in order to be included in the Agent. Find below how to install them with your Datadog Agent running as a binary on a host or as a container.
```

Comment:
Community developed integrations for the Datadog Agent are stored in the [Integrations-extra][1] Github repository. They are not packaged and built into the Datadog Agent, but can be installed as addons by following these instructions:"